tools/db_crashtest.py

```diff
@@ -236,6 +235,9 @@ def finalize_and_sanitize(src_params):
             dest_params["partition_filters"] = 0
         else:
             dest_params["use_block_based_filter"] = 0
+    if dest_params["index_type"] == 1 and \
+       dest_params.get("prefix_size", 7) == -1:
+       dest_params["index_type"] = 0       
```

Comment:
Instead of disabling index_type, how about setting prefix_size to 3?

Reply:
Right now we don't switch prefix size in the same run after crash. It doesn't mean RocksDB won't work with that, but we don't do that. Right now, the "--simple" run runs "prefix_size = -1", and other runs run default "prefix_size = 7". Changing it means that "simple" run is not "simple" anymore but switching between prefix mode on and off, which is the opposite to "simple".

Reply:
You did remind me that we should add changing prefix size in stress test, which should be supported. But I am not confident that it will pass...